botocore/client.py

```diff
@@ -319,6 +319,12 @@ def resolve(self, service_name, region_name=None, endpoint_url=None,
             return self._assume_endpoint(service_name, region_name,
                                          endpoint_url, is_secure)
 
+    def resolve_default(self, service_name, is_secure=True):
```

Comment:
Does it matter that you make a default here of ``True`` for ``is_secure`` when you actually do not use it in the ``redirect_create_bucket()``?